src/utils/normalize.js

```diff
@@ -0,0 +1,6 @@
+export function normalizeItemName(itemName) {
```

Comment:
Awesome! Thanks for adding this in. As a very small suggestion, I'd add a description comment above the helper function. so other members on the team are able to remember why this was added.